File: 프로그래머스/0/120849. 모음 제거/모음 제거.js
function solution(my_string) {
    var answer = '';
    
    const m_arr = ["a", "e", "i", "o", "u"];
    const m_set = new Set(m_arr);
    
    for(let i = 0 ; i < my_string.length ; i++){
        if(!m_set.has(my_string[i]))
            answer += my_string[i];
    }

    
    return answer;
}

File: 프로그래머스/0/120849. 모음 제거/모음 제거.java
import java.util.Set;
import java.util.stream.Collectors;

class Solution {
    public String solution(String my_string) {
        
        Set<String> vowels = Set.of("a", "e", "i", "o", "u");
        
        return my_string.chars()
                        .mapToObj(c -> (char) c)
                        .map(String::valueOf)
                        .filter(c -> !vowels.contains(c))
                        .collect(Collectors.joining());
        
    }
}